hack/components/bump-ovs-cni.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+
+source hack/components/yaml-utils.sh
+source hack/components/git-utils.sh
+
+echo Bumping ovs-cni
```

Comment:
some times you do use '' , see this file as well , worth to keep same direction imo